src/main.js

```diff
@@ -525,6 +526,44 @@ class MiniGraphCard extends LitElement {
       </svg>`;
   }
 
+  renderGridLines() {
+    const {
+      height, hours_to_show, grid_line_type,
+    } = this.config;
+
+    const containerWidth = 500;
+    let numLines;
+    let xRatio;
+
+    switch (grid_line_type) {
+      case 'hour':
+      default:
+        numLines = hours_to_show;
+        xRatio = containerWidth / numLines;
+        break;
+      case 'day':
+        numLines = Math.round(hours_to_show / 24);
+        xRatio = (containerWidth / numLines);
+        break;
+      case 'week':
+        numLines = Math.round(hours_to_show / 168);
+        xRatio = containerWidth / numLines;
+        break;
+    }
+    const lines = [];
+
+    for (let i = 0; i < numLines; i += 1) {
+      const x = xRatio * (i + 0.5);
+      if (i % 2 === 0) {
+        lines.push(svg`<line x1=${x} y1="0" x2=${x} y2=${height} stroke="lightgray" stroke-width="0.5" opacity="0.5"/>`);
```

Comment:
The `lightgray` color looks differently in light/dark themes.
Let's try using a standard `--divider-color` variable instead.
This variable currently is `rgba(225,225,225,0.12)` for a default dark theme and `rgba(0,0,0,0.12)` for a default light theme.
Using `rgb(from var(--divider-color) R G B /0.5)` will change an opacity to `0.5` from `0.12` for this CSS variable.
So, we may use
```
stroke=var(--divider-color)
```
for more frequent lines and use
```
stroke=rgb(from var(--divider-color) R G B /0.5)
```
for less frequent lines; and the `0.5` value should be revised.

Also, we can make it even more flexible:
```
stroke=var(--mini-graph-card-small-grid-color, var(--divider-color))
...
stroke=var(--mini-graph-card-large-grid-color, rgb(from var(--divider-color) R G B /0.5))
```
i.e. incorporate own CSS variables (which may be "enreached" in future releases).

Reply:
done